src/external-site/_vars-external-site.html

```diff
@@ -6,7 +6,7 @@
    ========================================================================== */
 
     // Find the external-urls in your local cfgov-refesh repo.
-    find . -name '*.html' ! -path './src/*' ! -path './node_modules/*' ! -path './_tests/*' \
+    find . -name '*.html' ! -path './*' ! -path '../node_modules/*' ! -path '../test/*' \
```

Comment:
It should run in the root of your cfgov-refresh directory. Best way to test it is to run the commands before/after the changes and compare.